src/components/Logo/Logo.tsx

```diff
@@ -9,10 +9,13 @@ const Logo = ({ type }: Props) => {
     const navigate = useNavigate();
     return (
         <span
-            style={{ fontSize: type == "big" ? "3rem" : "1.5rem" }}
+            style={{
```

Comment:
Dont use inline styles, use conditional classnames instead. For example type === "big" ? "largeFont" : "smallFont" and deal with styles in css file.